source/rust_verify/src/rust_to_vir_expr.rs

```diff
@@ -2371,6 +2408,25 @@ pub(crate) fn expr_to_vir_inner<'tcx>(
             }
             res => unsupported_err!(expr.span, format!("Path {:?}", res)),
         },
+        ExprKind::Path(qpath @ QPath::TypeRelative(_ty, _path_seg)) => {
+            let def = bctx.types.qpath_res(&qpath, expr.hir_id);
+            match def {
+                rustc_hir::def::Res::Def(_, def_id) => {
+                    let f_name = bctx.ctxt.tcx.def_path_str(def_id);
+                    if let Some(vir_expr) =
+                        int_intrinsic_constant_to_vir(expr.span, &expr_typ(), &f_name)
+                    {
+                        let mut erasure_info = bctx.ctxt.erasure_info.borrow_mut();
+                        erasure_info
+                            .resolved_calls
+                            .push((expr.span.data(), ResolvedCall::CompilableOperator));
+                        return Ok(vir_expr);
+                    }
+                }
+                _ => {}
```

Comment:
Should this be a panic? Or an unsupported feature?

Reply:
there's a panic on the line below

Reply:
Sorry, I misread.